core/bitpos.go

```diff
@@ -222,3 +222,47 @@ func getBitPosWithBitRange(byteSlice []byte, bitToFind byte, start, end int) int
 	// Bit not found in the range
 	return -1
 }
+
+// ByteSliceToObj converts a byte slice to an Obj of the specified type and encoding
```

Comment:
Actually the inverse functions of these were mentioned above in same file so that's why I thought to put it here.
if we are doing it like functionality wise like bitpos and bytearray then yeah we can put it in bytearray.go .

Reply:
Yes yes, i know. 
The earlier functions should have been in `bytearray.go `. 
However it feels a bit odd to have these common functions in an operation specific file.

Reply:
Acutually its my code. I should have added the functions to `bytearray.go` in the first place.  😦

Reply:
Oh okay, I have moved all those functions to `bytearray.go` .

Reply:
Thanks...